docs/computer/student_Reports/Acoustic Source - Mohd Hozaifa Khan.md

```diff
@@ -0,0 +1,161 @@
+
+---
+
+## Content
+
+1. [Introduction](#introduction)
+2. [Technologies Used](#technologies-used)
+3. [Methodology](#methodology)
+4. [Results and Performance](#results-and-performance)
+5. [Problems faced](#problems-faced)
+6. [Conclusion and Scope](#conclusion-and-scope) ( _proof read required!!_)
+7. [Further Developement](#further-developement)
+8. [Resources](#resources)
+
+---
+
+## Introduction
+
+Most of the EM Radiations especially RF waves are easily absorbed by water and hence modern localisation systems as well as communications system are rendered useless underwater. Underwater Vehicles need to utilize an alternate wave to perform these operations. One such and possibly the most feasible option is Sound. Moreover, Acoustics has already been used in SONARs and so there has been considerable development in its associated technology.
+
+For navigation there is always requirement of a good Positioning system, similar to GPS for terrestrial travels and navigation. One such basic approach is to develop a positioning system with respect to a stationary or terrestrial tracked sound source. This concept is also called Sound Source Localisation or Acoustic Source Localisation. There are many other applications of this technology as well.
+
+### SSL
+
+There have been various techniques and methods developed for SSL. Multilateration has already been in application for many years. Others methods are Beamforming, Eigen Vectors based, Neural Networks based, etc. To read in detail click [Here][review-paper] for a Review paper on SSL.
+
+### SSL: TDoA
+
+One of most used multilateration techniques is TDOA or Time Difference of Arrival method. It is similar to Inter-neural.  
+
+Consider the simplest case of Multilateration and TDoA for sound source localization.
+
+Take for instance a sound source S and two detectors or microphones M1 and M2. The simplest case is to determine on which side does the source lie with respect to the centre of line joining M1 and M2. Let's refer this point as O.
+
+The possible answers to the Questions _"Which side does the source S lie with respect to O, M1, M2?"_ can be **right** or **left**. A more accurate answer could the angle in degrees or radians subtended by source S at O from Line M1,M2. *Determining this angle in essence is a rudimentary form of Sound Source Localization*.
+
+Due to the unsymmetrical arrangement of Source, there will be a phase difference and hence a path difference in the waves recieved by M1 and M2. Calculating this path differennce can help calcuate the angle.
+
+Practically, instead of calculating path difference, it is prefered to determine Time difference between the arrivals of sound at the two detectors or micrphones. Hence, the technique is called *Time Difference of Arrival* method or **TDOA**
```

Comment:
```suggestion
Practically, instead of calculating path difference, it is preferred to determine Time difference between the arrivals of sound at the two detectors or microphones. Hence, the technique is called *Time Difference of Arrival* method or **TDOA**
```